packages/labs/react/src/create-component.ts

```diff
@@ -255,17 +264,19 @@ export const createComponent = <I extends HTMLElement, E extends Events = {}>(
           props[k === 'className' ? 'class' : k] = v;
         }
       }
-      return createElement(tagName, props);
+      return createElement<React.HTMLAttributes<I>, I>(tagName, props);
     }
   }
 
-  const ForwardedComponent = React.forwardRef(
-    (props?: UserProps, ref?: React.Ref<unknown>) =>
-      createElement(
-        ReactComponent,
-        {...props, __forwardedRef: ref} as ComponentProps,
-        props?.children
-      )
+  const ForwardedComponent: WrappedWebComponent<I, E> = React.forwardRef<
```

Comment:
Can you add the return type to `createComponent` on line 153?

Reply:
done!